src/main/webapp/i18n/en/guidedTourCourseExerciseOverview.json

```diff
@@ -3,7 +3,7 @@
         "courseExerciseOverview": {
             "installPrerequisites": {
                 "headline": "Install required tools",
-                "content": "Before we have a look at this course and the example exercise, we need to download and install a few software tools.<ul><li>Please download and install the newest <a href='https://www.oracle.com/java/technologies/javase-downloads.html' target='_blank' rel='noopener noreferrer'>Java JDK</a> (at least Java JDK 17). Make sure to choose JDK and <span class='red bold'>not</span> JRE.</li><li>Please download and install <a href='https://www.eclipse.org/downloads/packages/' target='_blank' rel='noopener noreferrer'>Eclipse IDE for Java Developers</a> (2021‑12 or later)</li><li>Please download and install <a href='https://www.sourcetreeapp.com' target='_blank' rel='noopener noreferrer'>SourceTree</a> with git (4.1.5 or later for macOS and 3.4.6 or later for Windows).</li></ul>",
+                "content": "Before we have a look at this course and the example exercise, we need to download and install a few software tools.<ul><li>Please download and install the newest <a href='https://www.oracle.com/java/technologies/javase-downloads.html' target='_blank' rel='noopener noreferrer'>Java JDK</a> (at least Java JDK 17). Make sure to choose JDK and <span class='red bold'>not</span> JRE.</li><li>Please download and install <a href='https://www.jetbrains.com/idea/download/' target='_blank' rel='noopener noreferrer'>IntelliJ</a> (2022.1.3 or later)</li><li>Please download and install <a href='https://www.sourcetreeapp.com' target='_blank' rel='noopener noreferrer'>SourceTree</a> with git (4.1.5 or later for macOS and 3.4.6 or later for Windows).</li></ul>",
```

Comment:
the guided tour still relies on Eclipse, so please revert this change and let's focus on the student exam guide in this PR